EnglishFilter/sections/antiadblock.txt

```diff
@@ -551,6 +551,7 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/76371
+animedao.to#%#//scriptlet("abort-current-inline-script", "script.onerror", "document.getElementById")
```

Comment:
I can't reproduce and do not see this script. https://github.com/AdguardTeam/AdguardFilters/issues/76371#issuecomment-791329281

Reply:
https://github.com/AdguardTeam/AdguardFilters/issues/76371#issuecomment-791332593